backend-services/README.md

```diff
@@ -0,0 +1,110 @@
+# Wallet Backend Services
+
+An HTTP server backend for the Nautilus Wallet powered by [FastAPI][fastapi] and
+the [uvicorn ASGI server][uvicorn].
+
+- [Setup](#setup)
+- [Quick Start](#quick-start)
+
+[fastapi]: https://fastapi.tiangolo.com/
+[uvicorn]: https://www.uvicorn.org/
+
+## Setup
+
+In order to proceed, one needs to have the `poetry` dependency management and
+packaging tool installed. Unless a recent version is available via your OS
+package manager, as it would be on Arch Linux and friends :), the recommended
+means of installing `poetry` is via the well-established `pipx` tool as
+described in [their documentation][pipx-install].
+
+Once `pipx` is installed, installing `poetry` is as simple as
+
+```shell
+pipx install poetry
```

Comment:
JP helped me debug that my poetry was still using Ubuntu's Python 3.8 and we fixed my env by specifying the python version when installing poetry.
```pipx install poetry --python python3.10```